README.md

```diff
@@ -1,15 +1,15 @@
 # purescript-orders
 
 [![Latest release](http://img.shields.io/github/release/purescript/purescript-orders.svg)](https://github.com/purescript/purescript-orders/releases)
-[![Build status](https://travis-ci.org/purescript/purescript-orders.svg?branch=master)](https://travis-ci.org/purescript/purescript-orders)
+[![Build status](https://github.com/purescript/purescript-orders/workflows/CI/badge.svg?branch=master)](https://github.com/purescript/purescript-orders/actions?query=workflow%3ACI+branch%3Amaster)
 [![Dependency status](https://img.shields.io/librariesio/github/purescript/purescript-orders.svg)](https://libraries.io/github/purescript/purescript-orders)
```

Comment:
Most core libraries don’t have this badge, should we remove it for consistency?

Reply:
Let's remove it.